frontend/store/companies/actions.js

```diff
@@ -0,0 +1,66 @@
+import types from "./mutation-types";
+
+export default {
+  async setCompanies({ commit }) {
+    await fetch("/api/faculties/wydzial-techniczny/companies.json")
+      .then((data) => data.json())
+      .then((data) => {
+        commit(types.SET_COMPANIES, data);
+      });
+  },
+  async setFilters({ commit }) {
+    await fetch("/api/faculties/wydzial-techniczny/filters.json")
+      .then((data) => data.json())
+      .then((data) => {
+        commit(types.SET_FILTERS, data);
+      });
+  },
+  setFaculties({ commit }) {
+    fetch("/api/faculties/faculties.json")
+      .then((data) => data.json())
+      .then((data) => {
+        commit(types.SET_FACULTIES, data);
+      });
+  },
+  setDocuments({ commit }) {
+    fetch("/api/faculties/wydzial-techniczny/documents/documents.json")
+      .then((data) => data.json())
+      .then((data) => {
+        commit(types.SET_DOCUMENTS, data);
+      });
+  },
+  setTags({ commit }) {
+    fetch("/api/faculties/wydzial-techniczny/filters.json")
+      .then((data) => data.json())
+      .then((data) => {
+        commit(types.SET_TAGS, data.tags);
+      });
```

Comment:
You're repeating the same code a lot of time, why not to create a wrapper function? eg. `fetchData`.

It would be something like this:

```js
export const fetchData = async (url) => {
    const response = await fetch(url);
    return response.json();
}
```

Reply:
Okay, already implemented.